editor/components/icon-button/index.js

```diff
@@ -14,9 +14,8 @@ function IconButton( { icon, children, label, className, ...additionalProps } )
 	const classes = classnames( 'editor-icon-button', className );
 
 	return (
-		<Button { ...additionalProps } aria-label={ label } className={ classes }>
+		<Button { ...additionalProps } aria-label={ label } className={ classes } data-level={ children ? children : null }>
```

Comment:
The children prop can not be used as an attribute because it can contain other tags (components).
I think this should be updated to use a `level` prop.
We also need a `children` prop like the one defined before. It's being used in the Block switcher PR https://github.com/WordPress/gutenberg/pull/429/files#diff-4d78380ac51947daf9b826f57a1684bbR58